config/credentials.yml.enc

```diff
@@ -1 +1 @@
-7KId8t/XynBn0te9eplC/CH+KabjZNO+nqtd0qehkkjFn6FWTTONUNmGiasZUqtMkSgUfoOVQJmoMfslG0pomifcQ/nN5Ry8Uzf7yA5W60JjU9RH05NN8N3QCD7j0W9WiLQs8Q5B419MylxnnqPc8vHzf/za63xG9r1PFdykyC5dM6qnRLHlGnIKum3P7b7pX759O2EdKyGFcOGl9DJGk1ueazzlKzx+mfZymgAss46WcCuOHIkP7jCPmIoPHAedqazPrnW9MIma2zmiIx2WhNPUw7NBSxFGfudhQMrkoVUPo+9CbaX6TxYLnark3Rpl9rsOcf7lAm1VHkP6eczWKl33KYsEHYmcZSVn6Cw0dy89zbQrMWTdU8VVU0t0qKMrhLcbjKuFkjdli01o+p+MKEEiZpdktAmB/vql--LwAQaesZM4ro5aUs--V8fqwqv5kNr9cBWaWDOc7g==
\ No newline at end of file
+PbdogYSdTI5ChMlQXWTLgCoD/EXdHIMZyRMNZ/a6xqjw15BqeNqhPdkURMA6XNssdBzzydL6muSjJHSqGPtT8kSftsFlr30CHpavqyyNowhWSzwf2AliGXTH3HEo/v5y0iq9izrWzobpXpcyHBXgLFOgLheXgelWSh+poFoNUZXTOT/gI4QgQnDkVUtTo4CZLQHhHnyWYnkoDKcL9J7c7MwMfmQxgtVuFxMD5SkhZMKceo8e8EDQ5BlT1AcgWagEvtKXfdmgydVOi6uW7szEvrNqx9Hyxx8P2gGjhH8mzf2KfcUNhgr2chjCdn/AwycpbibSsDD2mt7h9IwA7Wl3KKgbXemuIRbwt8r1XgPmT0FQH+odtEgBtAeujg0nxBaAMvWoe5iH89P7v+oRx0w2rtgNubqw1+tN4cOS--j33P9WMrXous267A--OljFhHTIfNS8x8pbIzMuFg==
```

Comment:
This should never ever be updated, FYI, but since we are starting from scratch, and don't have a DB initialized, it's ok. If it ever is updated, credentials won't work, and a new DB will have to be created.